utils/line-directive

```diff
@@ -61,7 +61,7 @@ line_pattern = re.compile(
 
 def _make_line_map(target_filename, stream=None):
     """
-    >>> from StringIO import StringIO
+    >>> from io import StringIO
```

Comment:
This will not work in Python 2, I'm afraid.

You can see in https://github.com/apple/swift/pull/23038/files#diff-cca357e5f9d41d8ba84f741ccf60fe55R64 how I solved it.